src/java.desktop/windows/native/libawt/windows/awt_PrintJob.cpp

```diff
@@ -57,6 +57,8 @@
  */
 #define ROUND_TO_INT(num)     ((int) floor((num) + 0.5))
 
+jfieldID AwtPrintDialog::pageID;
+
```

Comment:
where and why did this come from ?

Reply:
This came from below, all I did was move it up and out of the extern "C" block. This cannot be inside extern "C" because it is a static class member and has permanent C++ name mangling

Reply:
The question seems answered. Please resolve this discussion.